### PR TITLE
React native paper

### PR DESCRIPTION
**Pull request by diozenio**
# React native paper
- React native paper lib and styles implementation 

# Proposed Changes 
- implement react native paper themes
- add Poppins font
- temporary Home and CreatePost screens styles created

# Additional Info
- Metronic theme will be implemented on all pages in the next version

### CODE REVIEW

File: src/components/ButtonForm.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Button, withTheme } from "react-native-paper";
+
+const ButtonForm: React.FC = ({ children, ...rest }) => {
+  return (
+    <Button
+      mode="outlined"
+      {...rest}
+      style={{ backgroundColor: "#0097f7", paddingVertical: 5, marginTop: 15}}
```
Comment (Manuel-Antunes) [inline src/components/ButtonForm.tsx:9]: I'll let this PR on hold until you finish the styled component wrapping

File: src/components/Input.tsx [hunk 2]
```diff
@@ -1,91 +1,58 @@
...
+  }
+);
 
-export const LabelText = styled.Text`
-  color: ${(props) => props.theme.colors.info};
-  font-size: ${(props) => props.theme.fontSize.h3};
-  font-weight: bold;
-  margin-vertical: 15px;
-`;
+export default withTheme(Input);
```
Comment (Manuel-Antunes) [inline src/components/Input.tsx:58]: configure styled component here

File: App.tsx [hunk 3]
```diff
@@ -2,22 +2,50 @@ import { ThemeProvider } from "styled-components";
...
+    return <ActivityIndicator animating={true} color={"blue"} />;
+  } else {
+    return (
+      <AuthProvider>
+        <StatusBar
+          barStyle="light-content"
+          backgroundColor="transparent"
+          translucent
+        />
+        <ThemeProvider theme={theme}>
```
Comment (Manuel-Antunes) [inline App.tsx:38]: use the combined theme here too

File: src/screens/auth/SignIn.tsx [hunk 4]
```diff
@@ -36,48 +38,55 @@ const SignIn = () => {
...
+  color: ${(props) => props.theme.colors.text};
+  font-size: 18px;
   text-align: center;
 `;
 
 export const Submit = styled.TouchableOpacity`
   width: 100%;
-  padding: ${props => props.theme.spacing.p2};
-  background-color: ${props => props.theme.colors.submit};
+  padding: 30px;
```
Comment (Manuel-Antunes) [inline src/screens/auth/SignIn.tsx:84]: check it out a spacing property in react native paper

**Review by Manuel-Antunes — CHANGES_REQUESTED** [review]
Check my issues, add the screenshots, then request my review again

File: App.tsx (hunk 3 above)
Comment (diozenio) [reply App.tsx:38]: Changed to combined theme

File: src/screens/auth/SignIn.tsx (hunk 4 above)
Comment (diozenio) [reply src/screens/auth/SignIn.tsx:84]: I checked it out and it only has roundness property

File: src/components/Input.tsx (hunk 2 above)
Comment (diozenio) [reply src/components/Input.tsx:58]: Configured!! 👌

File: src/components/ButtonForm.tsx (hunk 1 above)
Comment (diozenio) [reply src/components/ButtonForm.tsx:9]: Styled component wrapping finished

**Pull request merged**